### PR TITLE
docs: refine terramate cloud drift show command usage

### PR DESCRIPTION
**Pull request by RocketRene**
## What this PR does / why we need it:
Refines the `terramate cloud drift show` command documentation for improved clarity and usability, aligning with documentation standards.

## Does this PR introduce a user-facing change?
Yes, updates documentation to clarify usage of the `terramate cloud drift show` command .

### CODE REVIEW

**Review by wmalik — DISMISSED** [review]

**Review by i4ki — APPROVED** [review]
LGTM

File: docs/cli/cmdline/cloud-drift-show.md [hunk 1]
```diff
@@ -9,4 +9,4 @@ The `cloud drift show` command shows any drift that has occurred in the current
 
 ## Usage
 
-`terramate -C <stack-directory> cloud drift show`
+`terramate [-C <stack-directory>] cloud drift show`
```
Comment (mariux) [inline docs/cli/cmdline/cloud-drift-show.md:12]: This is not optional as drift show does not run recursivly. it is only optional if you are in the stack.

**Review by mariux — CHANGES_REQUESTED** [review]

File: docs/cli/cmdline/cloud-drift-show.md (hunk 1 above)
Comment (mariux) [reply docs/cli/cmdline/cloud-drift-show.md:12]:
```
$ terramate cloud drift show
Error: No stack selected. Please enter a stack to show a potential drift.
```

Comment (i4ki) [reply docs/cli/cmdline/cloud-drift-show.md:12]: It's an optional flag, so it's correct to use `[-C <stack dir>]`. Maybe it needs additional explaining that you need either enter the directory or use `-C`.

Comment (mariux) [reply docs/cli/cmdline/cloud-drift-show.md:12]: At the moment it is required, we can not let users run into errors. Once we finished the feature, we can fix the docs

**Comment by mariux** [comment]
diff is empty, closing PR

**Pull request closed without merging**